### DISCUSSION TITLE
Show Edge Function invocations on project home stats

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#16978] [Contribute to Supabase]
We currently show these stats on the project home screen. Want to add a chart for Edge Function invocations to this.  

<img width="1290" alt="CleanShot 2023-07-03 at 10 21 39@2x" src="https://github.com/supabase/supabase/assets/105593/a94468cf-f7b7-463f-8adc-c2c144ebd958">

**Comment by ecktoteckto** [comment]
Hi @saltcod, I'd love to work on this one. Is that OK?

**Comment by ecktoteckto** [comment]
I've already had a look into it. The way this section is implemented makes the change actually straight forward I think. The only thing that's missing is the relevant data being returned from
`https://api.supabase.io/platform/projects/PROJECT-ID/analytics/endpoints/usage.api-counts?interval=hourly`

@saltcod Provided that you want me to work on this feature, could you help me out with the data?

Apart from data, is it supposed to look like this (apart from the icon)?
- Is the order of the panels supposed to be like this?
- Is the wording supposed to be like this?

<img width="1277" alt="supabase-15529" src="https://github.com/supabase/supabase/assets/31189692/c21b9baf-06da-4b42-9afd-4d9a89b53580">

**Comment by saltcod** [comment]
Yup that looks great! 

@phamhieu  could you have a look from the data side please?

**Comment by ecktoteckto** [comment]
Hi @phamhieu, just wanted to ask whether you've already had the chance to have a look into this. Thank you!
